From 9558797fb7bb019810d18f7acdde93adc35d02cc Mon Sep 17 00:00:00 2001
From: Morten Linderud <morten@linderud.pw>
Date: Sat, 2 Nov 2019 12:58:38 +0100
Subject: [PATCH 1/2] [Makefile] Fix docker documentation install and
 generation

The final versions of the documentation has been shifted from `docs/` to
`docs/build/man`. Most of the Makefile has been changed accordingly, but
the docker documentation generation was not.

Introduced by #4354

Signed-off-by: Morten Linderud <morten@linderud.pw>
---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 98b7bbdfd1..feb8e0ca3a 100644
--- a/Makefile
+++ b/Makefile
@@ -376,7 +376,7 @@ podman-remote-%-release:
 	$(MAKE) podman-remote-v$(RELEASE_NUMBER)-$*.zip
 
 docker-docs: docs
-	(cd docs; ./dckrman.sh *.1)
+	(cd docs; ./dckrman.sh ./build/man/*.1)
 
 changelog: ## Generate changelog
 	@echo "Creating changelog from $(CHANGELOG_BASE) to $(CHANGELOG_TARGET)"
@@ -425,7 +425,7 @@ install.cni:
 install.docker: docker-docs
 	install ${SELINUXOPT} -d -m 755 $(DESTDIR)$(BINDIR) $(DESTDIR)$(MANDIR)/man1
 	install ${SELINUXOPT} -m 755 docker $(DESTDIR)$(BINDIR)/docker
-	install ${SELINUXOPT} -m 644 docs/docker*.1 -t $(DESTDIR)$(MANDIR)/man1
+	install ${SELINUXOPT} -m 644 docs/build/man/docker*.1 -t $(DESTDIR)$(MANDIR)/man1
 
 install.systemd:
 	install ${SELINUXOPT} -m 755 -d ${DESTDIR}${SYSTEMDDIR}  ${DESTDIR}${USERSYSTEMDDIR} ${DESTDIR}${TMPFILESDIR}

From 079164c39193ee35641e2f4b680d8e69fe553625 Mon Sep 17 00:00:00 2001
From: Morten Linderud <morten@linderud.pw>
Date: Sat, 2 Nov 2019 13:48:47 +0100
Subject: [PATCH 2/2] [docs] Ensure we include section 5 documentation

Section 5 documentation is missing from the documentation restructure.

Signed-off-by: Morten Linderud <morten@linderud.pw>
---
 docs/{ => source/markdown}/containers-mounts.conf.5.md | 0
 docs/{ => source/markdown}/libpod.conf.5.md            | 0
 docs/{ => source/markdown}/podman-remote.conf.5.md     | 0
 3 files changed, 0 insertions(+), 0 deletions(-)
 rename docs/{ => source/markdown}/containers-mounts.conf.5.md (100%)
 rename docs/{ => source/markdown}/libpod.conf.5.md (100%)
 rename docs/{ => source/markdown}/podman-remote.conf.5.md (100%)

diff --git a/docs/containers-mounts.conf.5.md b/docs/source/markdown/containers-mounts.conf.5.md
similarity index 100%
rename from docs/containers-mounts.conf.5.md
rename to docs/source/markdown/containers-mounts.conf.5.md
diff --git a/docs/libpod.conf.5.md b/docs/source/markdown/libpod.conf.5.md
similarity index 100%
rename from docs/libpod.conf.5.md
rename to docs/source/markdown/libpod.conf.5.md
diff --git a/docs/podman-remote.conf.5.md b/docs/source/markdown/podman-remote.conf.5.md
similarity index 100%
rename from docs/podman-remote.conf.5.md
rename to docs/source/markdown/podman-remote.conf.5.md
